app/views/layouts/application.html.erb

```diff
@@ -104,15 +104,21 @@
       </div>
     <% end %>
     <%= yield %>
+    <% if FeatureFlag.enabled?(:runtime_banner) || ENV['E2E'] == 'true' %>
+      <div class="Browser-iOS">
+        <div class="runtime-banner">
+          <a href="https://udl.visualcosita.com/?r=<%= CGI.escape(URL.url("/r/mobile?deep_link=#{request.path}")) %>" class="flex items-center flex-1">
```

Comment:
Is this your domain? Will we eventually move the app behind it to a more "official" one?

Reply:
Yes, this is my domain. I wanted to ship this (still behind the FeatureFlag) so it's tangible and validated (make sure it's tested with the TestFlight build) while we wait for a final call and execution of the "official one".

Reply:
Non-blocking: should we extract this url into a url helper (maybe in the `URL` module? 🤔)

Reply:
I think it would be good to keep that logic outside the views and it also makes it easier to test there. Done ✅

Reply:
Update: Our "official" server is being used now 😄

Reply:
Great, thanks @fdoxyz!